py/server/deephaven/utils.py

```diff
@@ -0,0 +1,24 @@
+#
+#     Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+"""This module provides the helper and convenience functions that are not part of the Deephaven server but
+nonetheless uniquely useful to user applications built on top of Deephaven."""
+import gc
+
+import jpy
+
+from deephaven import DHError
+
+
+def garbage_collect() -> None:
+    """Runs full garbage collection in Python first and then requests the JVM to run its garbage collector. Since there
+    is no way to force the Java garbage collector to run, the effect of calling this function is non-deterministic.
+
+    Raises:
+        DHError
+    """
+    try:
+        gc.collect()
+        jpy.get_type("java.lang.System").gc()
```

Comment:
I've often found that it takes two calls to .gc() to free some of the python<->java references. I'm not sure if it can be executed twice in the same script invocation, or if they need to be in separate executions. We should be able to do a little testing; and if applicable, should invoke gc() here twice.

Reply:
Needing to call gc() twice makes sense and it is  allowed in my testing. I have also updated the doc string.